### PR TITLE
Dev example http jwt

### PR DESCRIPTION
**Pull request by otaviolcs3**

### CODE REVIEW

**Review by tjamir — CHANGES_REQUESTED** [review]
I don't know if it is my environment, but this example is not working here.
It gets coredumped (even when I fix the coredump error, I get a SSL error).

**Review by rodrigolc — APPROVED** [review]

File: svid/jwtsvid/tests/CMakeLists.txt [hunk 1]
```diff
@@ -20,4 +21,4 @@ target_link_libraries(check_jwtsvid svid ${CHECK_LIBRARIES}
   crypto
   pthread)
 
-add_test(check_jwtsvid check_jwtsvid)
+add_test(check_jwtsvid check_jwtsvid)
```
Comment (willallves) [inline svid/jwtsvid/tests/CMakeLists.txt:24]: add new line at end of file

File: workload/jwtsource.c [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #include "jwtsource.h"
 #include "jwtwatcher.h"
 
-void workloadapi_JWTSource_onJWTBundle_SetCallback(
-    jwtbundle_Set *jwt_set, void *args)
+void workloadapi_JWTSource_onJWTBundle_SetCallback(jwtbundle_Set *jwt_set,
+                                                   void *args)
```
Comment (willallves) [inline workload/jwtsource.c:5]: remove whitespace

File: workload/jwtsource.c [hunk 3]
```diff
@@ -94,7 +96,7 @@ err_t workloadapi_JWTSource_WaitUntilUpdated(workloadapi_JWTSource *source)
 }
 
 void workloadapi_JWTSource_applyJWTBundle_Set(workloadapi_JWTSource *source,
-                                             jwtbundle_Set *set)
+                                              jwtbundle_Set *set)
```
Comment (willallves) [inline workload/jwtsource.c:99]: remove whitespace

**Review by willallves — APPROVED** [review]

**Pull request merged**